game/game/playercontrol.cpp

```diff
@@ -855,13 +868,17 @@ void PlayerControl::processPickLock(Npc& pl, Interactive& inter, KeyCodec::Actio
   }
 
 void PlayerControl::processLadder(Npc& pl, Interactive& inter, KeyCodec::Action key) {
-  if(key==KeyCodec::Back) {
-    pl.setInteraction(nullptr);
+  if(key==KeyCodec::Back && !isPressed(KeyCodec::Forward)) {
+    inter.nextLadderMove(false,true);
```

Comment:
`false, true` - is not good from readability point of view.
It's probably better to handle key-repeat in `implMoveMobsi` routine.

Reply:
I kinda had that feeling but couldn't come up with something better, will change.